js/damage.js

```diff
@@ -725,6 +725,15 @@ function getDamageResult(attacker, defender, move, field) {
 		finalMods.push(0xC00);
 		description.defenderAbility = defender.ability;
 	}
+	if (attacker.hasItem("Metronome") && move.metronomeCount >= 1) {
+		var metronomeCount = Math.floor(move.metronomeCount);
+		if (metronomeCount <= 4) {
+			finalMods.push(0x1000 + metronomeCount * 0x333);
```

Comment:
Heh I was right, I told Antonio there was a much easier way to do this without looping

Reply:
I think that was he was trying to implement was different though, after updating what he did. Think he tried to add up the damage from every turn leading to the current turn.

Reply:
Yeah ik but his approach was still wrong